### PR TITLE
[TPC] further checkstyle improvements

### PR DESCRIPTION
**Pull request by pveentjer**
This one needs to be merged first:
https://github.com/hazelcast/hazelcast/pull/23395

### CODE REVIEW

File: hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/Socket.java [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
```
Comment (TomaszGaweda) [inline hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/Socket.java:2]: 2023 ;)

**Review by TomaszGaweda — APPROVED** [review]

File: hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/AsyncServerSocket.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ public long getAccepted() {
     }
 
     /**
-     * Gets the local address: the socket address that this channel's socket is bound to.
+     * Gets the local address of this ServerSocketChannel.
```
Comment (mdumandag) [inline hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/AsyncServerSocket.java:47]: The changes in this file basically revert the doc changes in the previous PR

Comment (pveentjer) [reply hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/AsyncServerSocket.java:47]: I must have messed up when rebasing.

Comment (pveentjer) [reply hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/AsyncServerSocket.java]: Fixed

File: hazelcast-tpc-engine/src/main/c/include/com_hazelcast_internal_tpc_iouring_CompletionQueue.h [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```
Comment (mdumandag) [inline hazelcast-tpc-engine/src/main/c/include/com_hazelcast_internal_tpc_iouring_CompletionQueue.h:1]: I think you have added these files by a mistake

**Review by mdumandag — APPROVED** [review]

**Pull request closed without merging**